### PR TITLE
Added github corner

### PR DESCRIPTION
**Pull request by SarveshLimaye**
Hii Mate. Added Github Corner on your efficient and useful website.

### CODE REVIEW

**Review by tusharnankani — CHANGES_REQUESTED** [review]
This is how it looks like right now:

image


And this after adding the corner:

image

The length of the input field has reduced, can you take a look?

**Comment by SarveshLimaye** [comment]
Yup sure

File: index.html [hunk 1]
```diff
@@ -28,19 +29,28 @@
...
         <form>
-            <input class="todo-input" type="text" placeholder="Add a task.">  
+            <input class="todo-input" type="text" placeholder="Add a task.">
             <button class="todo-btn" type="submit">I Got This!</button>
         </form>
     </div>
 
+    <!-- div for top left corner -->
+        <div class="version">
+                         <div class="demo version-section"><a href="#" class="github-corner">
```
Comment (tusharnankani) [inline index.html:41]: Can you link it to the repository?

**Review by tusharnankani — COMMENTED** [review]
Great work!

File: index.html (hunk 1 above)
Comment (SarveshLimaye) [reply index.html:41]: Could you guide me to do it

Comment (tusharnankani) [reply index.html:41]: > Could you guide me to do it. As i am a very new user of github

I meant, replace `#` with the URL of the ToDoList repository link, on line number 41.

Comment (tusharnankani) [reply index.html]: Nevermind, I fixed it.

**Comment by tusharnankani** [comment]
Good work @SarveshLimaye 👍🏻

**Pull request merged**